A better README (#22)
githubtraining/write-a-ll-course: README.md

> Author your own course on GitHub Learning Lab in just a few minutes! 

Reviewer: ```suggestion
Learning Lab helps people build real world skills through hands-on activities. This course streamlines the course building process, and gives you all the tools you need to help others learn. 
```

After completing this course, you'll be able to:

> - identify the components of a course on GitHub Learning Lab

Reviewer: ```suggestion
- identify the structure of a Learning Lab course
```

> - create course metadata

Reviewer: ```suggestion
- write metadata for machine readability 
```
^ I'm not sure this suggestion is a great solution, but do you think the user needs any extra understanding for this step? I could be overthinking it.

> Register for this course to get the starter files needed to build your Learning Lab course, or complete the course steps for a fully guided experience. If you choose to complete the course, you'll be able to:

Reviewer: Minor nit, but do you think it would be organized better to move "Register for this course to get the starter files needed to build your Learning Lab course, or complete the course steps for a fully guided experience." into the first paragraph?

Reviewer: nvm
